core-blocks/latest-comments/edit.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import {
+	PanelBody,
+	RangeControl,
+	ToggleControl,
+	ServerSideRender,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import {
+	InspectorControls,
+	BlockAlignmentToolbar,
+	BlockControls,
+} from '@wordpress/editor';
+
+/**
+ * Internal dependencies.
+ */
+import './editor.scss';
+
+const MIN_COMMENTS = 1;
+const MAX_COMMENTS = 100;
+
+class LatestComments extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.setAlignment = this.setAlignment.bind( this );
+		this.setCommentsToShow = this.setCommentsToShow.bind( this );
+		this.toggleAttribute = this.toggleAttribute.bind( this );
+	}
+
+	toggleAttribute( propName ) {
+		return () => {
+			const value = this.props.attributes[ propName ];
+			const { setAttributes } = this.props;
+
+			setAttributes( { [ propName ]: ! value } );
+		};
+	}
+
+	setAlignment( nextAlign ) {
+		const { setAttributes } = this.props;
+
+		setAttributes( { align: nextAlign } );
+	}
+
+	setCommentsToShow( commentsToShow ) {
+		const { setAttributes } = this.props;
+
+		setAttributes( { commentsToShow: parseInt( commentsToShow, 10 ) || 0 } );
+	}
+
+	render() {
+		const {
+			align,
+			commentsToShow,
+			displayAvatar,
+			displayExcerpt,
+			displayTimestamp,
+		} = this.props.attributes;
+
+		return (
+			<Fragment>
+				<BlockControls>
+					<BlockAlignmentToolbar
+						value={ align }
+						onChange={ this.setAlignment }
+					/>
+				</BlockControls>
+				<InspectorControls>
+					<PanelBody title={ __( 'Latest Comments Settings' ) }>
+						<ToggleControl
+							label={ __( 'Display avatar' ) }
+							checked={ displayAvatar }
+							onChange={ this.toggleAttribute( 'displayAvatar' ) }
+						/>
+						<ToggleControl
+							label={ __( 'Display date/time' ) }
```

Comment:
I only see the date, no time
![bildschirmfoto 2018-07-13 um 14 33 29](https://user-images.githubusercontent.com/695201/42691810-bceff520-86a9-11e8-999d-dbaffaeba4e8.png)

Reply:
Huh, seems right. It used to say "timestamp" which is definitely wrong, but maybe it should be using a better date function. I'll check it out...